posthog/api/alert.py

```diff
@@ -246,13 +247,35 @@ def validate(self, attrs):
         if attrs.get("insight") and attrs["insight"].team.id != self.context["team_id"]:
             raise ValidationError({"insight": ["This insight does not belong to your team."]})
 
-        if attrs.get("enabled") is not False and (
-            AlertConfiguration.objects.filter(team_id=self.context["team_id"], enabled=True).count()
-            >= AlertConfiguration.ALERTS_PER_TEAM
-        ):
-            raise ValidationError(
-                {"alert": [f"Your team has reached the limit of {AlertConfiguration.ALERTS_PER_TEAM} enabled alerts."]}
-            )
+        user_org = self.context["request"].user.organization
+
+        has_alerts_feature = user_org.is_feature_available(AvailableFeature.ORGANIZATIONS_PROJECTS)
+
+        allowed_alerts_count = next(
+            (
+                feature.get("limit")
+                for feature in user_org.available_product_features or []
+                if feature.get("key") == AvailableFeature.ALERTS
+            ),
+            None,
+        )
+
```

Comment:
A little confused - we use ORGANIZATIONS_PROJECTS to determine if they have the alerts feature, but there's another AvailableFeature called ALERTS? What's the difference?

Reply:
good catch! Forgot update after revert 🤦